client/my-sites/stats/activity-log/index.jsx

```diff
@@ -193,14 +218,16 @@ class ActivityLog extends Component {
 			startDate,
 		} = this.props;
 
-		const startOfMonth = moment( startDate ).startOf( 'month' ),
-			startOfMonthMs = startOfMonth.valueOf(),
-			endOfMonthMs = moment( startDate ).endOf( 'month' ).valueOf();
-		const filteredLogs = filter( logs, obj => startOfMonthMs <= obj.ts_site && obj.ts_site <= endOfMonthMs );
-		const logsGroupedByDate = map(
+		const siteOffset = this.getSiteOffsetFunc();
+
+		const startOfMonthMs = siteOffset( moment( startDate ).startOf( 'month' ) ).valueOf();
+		const endOfMonthMs = siteOffset( moment( startDate ).endOf( 'month' ) ).valueOf();
```

Comment:
I've been back and forth on this in my head for a few days, but I have concluded that for comparisons:

site-time in site-month === utc-time in utc-month

because everything is being offset by the same amount.